.github/workflows/performance.yml

```
@@ -95,3 +95,11 @@ jobs:
                   name: failures-artifacts
                   path: ./__test-results/artifacts
                   if-no-files-found: ignore
+
+            - name: Run Reassure performance tests
```

IMO Reassure is an implementation detail. Using the term "mobile" may be a more helpful description for others observing this CI task. WDYT? 

```suggestion
            - name: Run mobile performance tests
```